hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -135,6 +135,14 @@ enum Type {
   SnapshotPurge = 118;
   RecoverLease = 119;
   SetTimes = 120;
+  SetSafeMode = 121;
```

Comment:
@jojochuang , 121 is already taken in the master branch.  Please sync this with the master branch.
```
<<<<<<< HEAD
  RefetchSecretKey = 121;
=======
  SetSafeMode = 121;
>>>>>>> jojochuang/HDDS-8436
```